types/lodash/lodash-tests.ts

```diff
@@ -11266,6 +11266,22 @@ namespace TestGet {
     _.get([], Symbol.iterator);
     _.get([], [Symbol.iterator]);
 
+    _.get('abc', 4); // $ExpectType string
+    _('abc').get(4); // $ExpectType string
+    _.chain('abc').get(4); // $ExpectType LoDashExplicitWrapper<string>
```

Comment:
Seems a little strange to have the test get an out-of-bounds index. Why not 1 or 2?